src/ResourceManager/RedisCache/Commands.RedisCache/Models/RedisCacheAttributesWithAccessKeys.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.Azure.Commands.RedisCache.Models
 
     public class RedisCacheAttributesWithAccessKeys : RedisCacheAttributes
     {
-        public RedisCacheAttributesWithAccessKeys(RedisResourceWithAccessKey cache, string resourceGroupName)
```

Comment:
This is just return type for Set and New operation so customer won't have requirement to create them actually.